src/otx/v2/adapters/torch/mmengine/engine.py

```diff
@@ -264,15 +274,15 @@ def train(
                 work_dir=str(target_folder),
                 experiment_name="otx_train",
                 cfg=Config({}),  # To prevent unnecessary dumps.
-                **self.config,
+                **config,
             )
         if checkpoint is not None:
             self.runner.load_checkpoint(checkpoint)
-        self.dumped_config = dump_lazy_config(config=self.config, scope=self.registry.name)
+        self.dumped_config = dump_lazy_config(config=config, scope=self.registry.name)
```

Comment:
when `dumped_config` is used? This looks like used and updated during export operation.

Reply:
This is a temporary way of storing the model's config, which we'll fix later in a cleaner way.

Reply:
alright.